portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase

```diff
@@ -532,7 +524,7 @@ definition {
 		}
 
 		task ("Wait for sending requests") {
-			Pause(locator1 = "10000");
+			ACUtils.harRecordingPause();
 		}
```

Comment:
Hey @calvinctvo, I think for this situation of waiting for events to appear, you should create a macro with a clearer name for it. This macro would only have a 5 seconds pause and a comment justifying it and an LRAC ticket for more information.

**ex: WaitSendingRequests or WaitRequests**